packages/profile-sync-controller/CHANGELOG.md

```diff
@@ -7,6 +7,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.5]
+
+### Added
+
+- Add and export object `USER_STORAGE_SCHEMA`, function `getFeatureAndKeyFromPath`, and type `UserStoragePath` ([#4543](https://github.com/MetaMask/core/pull/4543))
+
+### Changed
+
+- **BREAKING:** Enforce `UserStorageSchema` usage in all functions that get or set user storage ([#4543](https://github.com/MetaMask/core/pull/4543))
+  - Keeps user storage entries consistent, and improves DX for consumers of user storage.
+  - **BREAKING:** Remove `entryKey` function parameter from `performGetStorage` and `performSetStorage` methods of `UserStorageController`, and replace with `path` parameter of type `UserStoragePath`.
+  - **BREAKING:** Remove `entryKey` function parameter from `createEntryPath`, and replace with `path` parameter of type `UserStoragePath`.
+  - **BREAKING:** Remove `entryKey` property from type `UserStorageOptions`, and replace with `path` property of type `UserStoragePath`.
+- Upgrade TypeScript to v5.0 and set `moduleResolution` option to `Node16` ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/base-controller` from `^6.0.0` to `^6.0.2` ([#4517](https://github.com/MetaMask/core/pull/4517), [#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/snaps-controllers` from `^8.1.1` to `^9.3.1` ([#3645](https://github.com/MetaMask/core/pull/3645))
+  - Bump peerDependency `@metamask/snaps-controllers` from `^8.1.1` to `^9.3.0`
```

Comment:
If we are bumping a peer dependency then this is a breaking change as the client will also need to upgrade this dependency. Should we bump this package to 0.2.0?

Reply:
@Prithpal-Sooriya How would you feel about using a minor version bump to indicate that there are breaking changes in the release?

Reply:
Upgraded both `{notifications-services,profile-sync}-controller` to `0.2.0`:  011b6b68eaa28b650edce0f93a93a209cfbcd2db

@Prithpal-Sooriya Let me know if this goes against the team's intentions. I can simply revert that commit.